packages/terra-compact-interactive-list/src/utils/keyHandlerUtils.js

```diff
@@ -131,6 +131,29 @@ const getFirstSemanticRowIndexInVisualRow = (rowsLength, numberOfColumns, flowHo
   return firstItemInVisualRow;
 };
 
+/**
+   * Finds the last semantic row index in a visual row, where the given row is located
+   * @param {number} rowsLength - a total number of seamntic rows in the list.
+   * @param {number} numberOfColumns - a number of visual columns.
+   * @param {boolean} flowHorizontally - sematic rows horizontal flow direction
+   * @param {number} row - an index of the currently focused semantic row.
+   * @returns - the index of the last semantic row in the same visual row as currently focused row.
+   */
+const getLastSemanticRowIndexInVisualRow = (rowsLength, numberOfColumns, flowHorizontally, row) => {
```

Comment:
This functionality wasn't added in this change, it existed before. I just had to move it up in order to use in another method.